src/components/controls/Controls.js

```diff
@@ -0,0 +1,26 @@
+import "./controls.css";
+import menu from "../../img/menu.svg";
+import search from "../../img/search.svg";
+
+export function Controls() {
+  function showOptions() {
+    const searchBackground = document.querySelector(".options-background");
+    searchBackground.classList.remove("hide-options");
+  }
+
+  function showSearch() {
+    const searchBackground = document.querySelector(".search-background");
+    searchBackground.classList.remove("hide-search");
+  }
+  return (
+    <>
```

Comment:
remove the <> and add a proper container, this makes for better visualise the whole component in the inspector